Mastodon/Scene/HomeTimeline/HomeTimelineViewController.swift

```diff
@@ -140,10 +140,89 @@ final class HomeTimelineViewController: UIViewController, NeedsDependency, Media
             case .home:
                 showLocalTimelineAction.state = .off
                 showFollowingAction.state = .on
+            case .list:
+                showLocalTimelineAction.state = .off
+                showFollowingAction.state = .off
+            case .hashtag:
+                showLocalTimelineAction.state = .off
+                showFollowingAction.state = .off
+            }
+        }
+        
+        let listsSubmenu = UIDeferredMenuElement.uncached { [weak self] callback in
+            guard let self else { return callback([]) }
+            
+            Task { @MainActor in
+                let lists = (try? await Mastodon.API.Lists.getLists(
```

Comment:
If there aren't any lists (or I don't follow any hashtags) but tap on "Lists"/"Followed Hashtags", the submenu disappears right away and this feels kind of broken

Reply:
This makes sense, I looked at how it's done on android and there an empty view is shown, I'll add something similar.

Reply:
This was the most straight forward way to do:

<img width="231" alt="Bildschirmfoto 2024-07-17 um 13 17 54" src="https://github.com/user-attachments/assets/9296bcb9-461d-4ac2-9393-136905d189e4">
<img width="228" alt="Bildschirmfoto 2024-07-17 um 13 17 59" src="https://github.com/user-attachments/assets/ddf4cf36-0363-4404-8d77-0b775683a615">